contrib/pg_stat_statements/expected/pg_stat_statements.out

```diff
@@ -182,32 +183,37 @@ SELECT * FROM test ORDER BY a;
 (12 rows)
 
 -- SELECT with IN clause
-SELECT * FROM test WHERE a IN (1, 2, 3, 4, 5);
+SELECT * FROM test WHERE a IN (1, 2, 3, 4, 5) ORDER BY a, b;
  a |          b           
 ---+----------------------
  1 | 111                 
+ 1 | a                   
  2 | 222                 
+ 2 | b                   
  3 | 333                 
+ 3 | c                   
  4 | 444                 
  5 | 555                 
- 1 | a                   
- 2 | b                   
- 3 | c                   
 (8 rows)
 
-SELECT query, calls, rows FROM pg_stat_statements ORDER BY query COLLATE "C";
-                            query                            | calls | rows 
--------------------------------------------------------------+-------+------
- DELETE FROM test WHERE a > $1                               |     1 |    1
- INSERT INTO test (a, b) VALUES ($1, $2), ($3, $4), ($5, $6) |     1 |    3
- INSERT INTO test VALUES(generate_series($1, $2), $3)        |     1 |   10
- SELECT * FROM test ORDER BY a                               |     1 |   12
- SELECT * FROM test WHERE a > $1 ORDER BY a                  |     2 |    4
- SELECT * FROM test WHERE a IN ($1, $2, $3, $4, $5)          |     1 |    8
- SELECT pg_stat_statements_reset()                           |     1 |    1
- UPDATE test SET b = $1 WHERE a = $2                         |     6 |    6
- UPDATE test SET b = $1 WHERE a > $2                         |     1 |    3
-(9 rows)
+SELECT count(*), query, sum(calls) calls, sum(rows) rows FROM pg_stat_statements GROUP BY query ORDER BY query COLLATE "C";
+ count |                              query                               | calls | rows 
+-------+------------------------------------------------------------------+-------+------
+     4 | DELETE FROM test WHERE a > $1                                    |     4 |    1
+     4 | INSERT INTO test (a, b) VALUES ($1, $2), ($3, $4), ($5, $6)      |     5 |    3
+     4 | INSERT INTO test VALUES(generate_series($1, $2), $3)             |     5 |   10
```

Comment:
Why do we get 5 calls here?

Reply:
```
 Insert on test  (cost=0.00..0.20 rows=3 width=88) (never executed)
   ->  Redistribute Motion 1:3  (slice1; segments: 1)  (cost=0.00..0.20 rows=3 width=88) (actual time=1.296..1.299 rows=5 loops=1)
         Hash Key: (generate_series(1, 10))
         ->  ProjectSet  (cost=0.00..0.07 rows=10 width=88) (actual time=0.010..0.013 rows=10 loops=1)
               ->  Result  (cost=0.00..0.01 rows=1 width=0) (actual time=0.005..0.005 rows=1 loops=1)
 Planning Time: 0.406 ms
   (slice0)    Executor memory: 36K bytes avg x 3 workers, 36K bytes max (seg0).
   (slice1)    Executor memory: 42K bytes (seg1).
 Memory used:  128000kB
```

One of segments additionally executes slice1 (e.g. generate_series)